### ISSUE TITLE
Invalid inline view markup

### ISSUE THREAD

**Issue by netcitylife** [jmzagorski/aurelia-fontawesome#12]
**I'm submitting a bug report**

* **Library Version:**
0.2.0

* **Browser:**
all

**Current behavior:**
Inline view code looks like this:
```
<template>
    <div id="svg-holder" innerhtml.bind="_iconhtml"></div>
</template><tempate>
```

**What is the expected behavior?**
First of all - id attribute should be removed because multiple same IDs would be present on page when using multiple icons. Second - `<tempate>` at the end.

**What is the motivation / use case for changing the behavior?**
I would suggest to change template to, it doesn't create extra markup and works perfectly:
```
<template innerhtml.bind="_iconhtml"></template>
```

**Comment by jmzagorski** [comment]
That was an oversight on my part, not sure why I put that id in there to begin with. Will fix.

**Comment by jmzagorski** [comment]
Fixed and published to npm as 0.2.1. Thanks for finding this mistake!
